packages/icons/icons.json

```
@@ -1430,49 +1430,49 @@
     {
         "displayName": "Stop",
         "iconName": "stop",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, box, square, error, not possible, blocked, black square, shape",
         "group": "media",
         "codepoint": 62161
     },
     {
         "displayName": "Play",
         "iconName": "play",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, triangle, right triangle, start",
         "group": "media",
         "codepoint": 62072
     },
     {
         "displayName": "Fast forward",
         "iconName": "fast-forward",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, ff, forward, double arrow",
         "group": "media",
         "codepoint": 61861
     },
     {
         "displayName": "Step forward",
         "iconName": "step-forward",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, play, fast forward, next, skip ahead, skip",
         "group": "media",
         "codepoint": 62160
     },
     {
         "displayName": "Eject",
         "iconName": "eject",
-        "tags": "player, media, controls, digital, analogue, film, audio, video",
+        "tags": "player, media, controls, digital, analogue, film, audio, video, triangle, rectangle, up triangle, up, arrow, upload",
```

```suggestion
        "tags": "player, media, controls, digital, analogue, film, audio, video",
```

I don't think any of these actually help, there other better triangle/arrow icons